identify tool child feature (#6322)
qgis/QGIS-Documentation: docs/user_manual/introduction/general_tools.rst

* **Data attributes**: This is the list of attribute fields and values for the
  feature that has been clicked.
* information about the related child feature if you defined a relation as described
  in :ref:`vector_relations´:
  
  * the name of the relation
  * the entry in reference field, e.g. the name of the related child feature

>   * **Actions**: Also for the child feature you can define actions
>     in the layer's properties dialog (see :ref:`actions_menu`) and the default
>     action is ``View feature form``.

Reviewer: You do not define the actions here, they are just listed if of canvas scope afaict
```suggestion
  * **Actions**: lists actions defined in the layer's properties dialog (see :ref:`actions_menu`)
    and the default action is ``View feature form``.
```
